webapp/components/LoginForm/LoginForm.vue

```diff
@@ -20,15 +20,28 @@
           name="email"
           icon="envelope"
         />
-        <ds-input
-          v-model="form.password"
-          :disabled="pending"
-          :placeholder="$t('login.password')"
-          icon="lock"
-          icon-right="question-circle"
-          name="password"
-          type="password"
-        />
+        <div class="password-wrapper">
+          <ds-input
+            v-model="form.password"
+            :disabled="pending"
+            :placeholder="$t('login.password')"
+            icon="lock"
+            name="password"
+            class="password-field"
+            :type="showPassword ? 'text' : 'password'"
+          />
+          <a
+            class="click-wrapper"
+            @mousedown="
+              (event) => {
+                showPassword = !showPassword
+                event.preventDefault()
+              }
+            "
+          >
+            <base-icon class="toggle-icon" :name="showPassword ? 'eye-slash' : 'eye'" />
```

Comment:
The cursor should change on hover that the user can see that the icon is clickable

Reply:
Thanks for the feedback. I will adjust those. I also noticed there is a slight issue with where the icon is clickable. I will work to adjust that as well.

Reply:
Yes, I noticed this as well, but have no explanation so far.